rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -141,9 +132,6 @@ void eventLogger_logReleaseBtcRequested_after_rskip_146() throws IOException {
 
     @Test
     void eventLogger_logReleaseBtcRequested_after_rskip_146_185() throws IOException {
-        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
-        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(true);
-
         bridgeSupport.releaseBtc(releaseTx);
```

Comment:
I think we may rename this `bridgeSupport` instance to a better name, that indicates it was created with all the activations active. Wdyt?

Reply:
It's overriding the value defined above, changing it impacts a few places. I don't want to get into that right now